ansible_collections/arista/avd/roles/eos_designs/python_modules/connected_endpoints/port_channel_interfaces.py

```diff
@@ -203,18 +202,13 @@ def _add_port_channel_if_not_duplicate(self, candidate_port_channel_config, port
 
         If 1 is False for every port-channel in the port_channel_interfaces, it is a new port-channel and it is appended to the list.
         """
-        if (matching_port_channel_config := get_item(port_channel_interfaces, "name", candidate_port_channel_config["name"])) is None:
-            # No port_channel_interface found with the same name in port_channel_interfaces
-            # append to the list and return
-            port_channel_interfaces.append(candidate_port_channel_config)
-            return
-
-        if matching_port_channel_config != candidate_port_channel_config:
-            # Found duplicate name with different generated configs
-            raise AristaAvdError(
-                f"Duplicate port-channel name {candidate_port_channel_config['name']} with conflicting configurations found while generating port-channels for"
-                " connected-endpoints or network-ports"
-            )
+        append_if_not_duplicate(
```

Comment:
do you still need the wrapper function `_add_port_channel_if_not_duplicate` ? it looks like we could remove it and just call `append_if_not_duplicate` in the main code?

Reply:
fixed